src/index.ts

```diff
@@ -72,15 +75,24 @@ export class Resolver {
    * @returns {Promise<AccountId>}
    */
   public async resolveSLD(domain: string): Promise<string | undefined> {
+    if (await this._udResolver.isSupportedDomain(domain)) {
+      try {
+      const udResult = await this._udResolver.addr(domain, 'HBAR')
+      return udResult;
+      } catch (error) {
+        console.log(error);
+        return Promise.resolve(undefined);
+      }
```

Comment:
this try catch doesn't have correct formatting additionally further it is nested in an `if` statement which may have a few negative consequences but most notably it will be hard to maintain. I would suggest breaking this logic out into its own function or refactoring this whole function

Reply:
Fixed, but not exactly the way you want it I think. Let me know how you feel about it.